tests/tests_integration/conftest.py

```diff
@@ -4,21 +4,21 @@
 import pytest
 
 from cognite.client import ClientConfig, CogniteClient
-from cognite.client.credentials import OAuthClientCredentials, OAuthInteractive
+from cognite.client.beta import CogniteClient as CogniteBetaClient
+from cognite.client.credentials import CredentialProvider, OAuthClientCredentials, OAuthInteractive
 
 
-@pytest.fixture(scope="session")
-def cognite_client() -> CogniteClient:
+def _get_credentials() -> CredentialProvider:
```

Comment:
Let's make this a session-scoped fixture aswell so we don't instantiate the credential providers multiple times (the credential provider instantiation has side effects).

Reply:
Tried it in this commit https://github.com/cognitedata/cognite-sdk-python/pull/1029/commits/3c9f48809dad6ab1d527acb323ef6a4b6d703085, seems like it doesn't work